frontend/__tests__/containers/ReportStep/ReportStep.test.tsx

```diff
@@ -685,4 +687,63 @@ describe('Report Step', () => {
       expect(closeNotification).toHaveBeenCalledTimes(1);
     });
   });
+
+  describe('error notification', () => {
```

Comment:
There is 2 `describe('error notification')` in this file.

Reply:
resolved